src/test/java/org/fundacionjala/coding/TestCaseRental.java

```diff
@@ -0,0 +1,194 @@
+package org.fundacionjala.coding;
+
+
+import org.fundacionjala.coding.dennis.MovieChildren;
+import org.fundacionjala.coding.dennis.Movie;
+import org.fundacionjala.coding.dennis.MovieNewRelease;
+import org.fundacionjala.coding.dennis.PriceCode;
+import org.fundacionjala.coding.dennis.Rental;
+import org.fundacionjala.coding.dennis.Customer;
+import org.fundacionjala.coding.dennis.MovieRegular;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ *
+ */
+public class TestCaseRental {
+
+    public static final double LIMIT = 0.01;
+    public static final int DAYSRENTED = 3;
+    public static final int DAYSRENTED4 = 4;
+
+    /**
+     *
+     */
+    @Test
+    public void testPriceMovieRentalAtmPass() {
+        Rental rent = new Rental(new MovieNewRelease("Isle of dogs"), DAYSRENTED);
+        final float expected = 9.0f;
+        assertEquals(expected, rent.calculatePriceMovie(), LIMIT);
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void testFrequentRenterPointsAtmPass() {
+        Rental rent = new Rental(new MovieNewRelease("Infinity War"), DAYSRENTED);
+        int expected = 2;
+        assertEquals(expected, rent.calculateFrequentRenterPoints());
+    }
+
+    /**
+     *
+     */
+    @Test
+    public void testGetPriceCodeMoviePass() {
+        Movie mov = new MovieNewRelease("Annihilation");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Annihilation" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)